.github/workflows/nightly.yml

```diff
@@ -2,8 +2,8 @@ name: Nightly Build
 
 on:
   workflow_dispatch:
```

Comment:
all of these have workflow_dispatch; 

This is required for recovery purposes, correct?

Reply:
This allows us to run the jobs manually if we like, which provides parity with codefresh